app/js/app_routes.js

```diff
@@ -78,12 +78,14 @@ app.config(["$routeProvider", '$locationProvider', function ($routeProvider, $lo
 
         when('/workflows',                              { "redirectTo": '/clearing-house/records/failed-workflows'}).
         
-        when('/clearing-house'                              ,{ ...mapView(vueViewWrapper)    , label:'Clearing-House Management',   resolveUser : true, resolve : { ...routeTemplates.views_chm_index            , securized : securize([...chmAdminRoles, 'Administrator'], null, true) }}).
+        // clearing-house/:environment? route is interfering with specific paths
```

Comment:
can you please explain what is the conflict?

Reply:
Angular will use the first matching route, potentially leading to unwanted behavior or conflicts in resolving the correct view. we need to reorder Routes by Placeing the most specific routes before the more general ones. Since '/clearing-house/:environment?' is more general, it should be placed after the other routes.